packages/string-templates/src/index.ts

```diff
@@ -12,11 +12,12 @@ import {
 } from "./utilities"
 import { convertHBSBlock } from "./conversion"
 import { setJSRunner, removeJSRunner } from "./helpers/javascript"
-import { helpersToRemoveForJs } from "./helpers/list"
```

Comment:
What are the changes here needed for out of interest?

Reply:
We were importing just to export, renaming it to `_whatever`. This is not actually needed, but it slipped while refactoring `string-templates` to esm